README.md

```diff
@@ -9,6 +9,50 @@ Authors: [@TheThirdOne](https://github.com/TheThirdOne),
 
 ![](stark.gif)
 
+-------------------------------------------------------------------------------
+
+## Objective
+
+The objective of Maestro is to allow a beginner to use services like 
+[Twilio](https://www.twilio.com/products) without being slowed by the details
+of how to implement the API. This will help beginners build more/better 
+applications which will help encourage them to continue learning.
+
+## Usage
+
+The JS file containing the library is found at `examplehost.com/static/baton.js`
```

Comment:
Perhaps just say `/static/baton.js`? I imagine `examplehost.com` may be a bit confusing.